### ISSUE TITLE
Cannot access to css files in phar archive

### ISSUE THREAD

**Issue by vonglasow** [hoaproject/Kitab#8]
```
php ./Kitab.phar compile module/Core
Hoa\File\File::_open(): (2) Failed to open stream phar:///var/www/project/Kitab.phar/src/Compiler/Target/Html/Template/Public/css/main.css
.
in phar:///var/www/project/Kitab.phar/vendor/hoa/file/File.php at line 208.
```

**Comment by Hywan** [comment]
Happens only with Kitab.phar.

**Comment by Hywan** [comment]
When the `Template` directory is copied, the `Hoa\File\Directory::copy` method is used. This latter open files in read/write mode. This is forbidden in a PHAR. That's why it fails.

**Comment by Hywan** [comment]
https://github.com/hoaproject/File/pull/27 must be merged and released to fix this issue. Another issue happens however:

```
module.js:472
    throw err;
    ^

Error: Cannot find module '/Users/hywan/Development/Kitab/phar:/Users/hywan/Development/Kitab/Kitab.phar/src/Compiler/Target/Html/Javascript/search-build-index.js'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:427:7)
    at startup (bootstrap_node.js:148:9)
    at bootstrap_node.js:542:3
/var/folders/m0/4f6b8rhj4gnb_29c_gz1xq0m0000gn/T/Kitab.html.output/index.html
```

I will fix this in this issue.

**Comment by Hywan** [comment]
Fixed in 0.6.
